myuw/static/js/card/schedule/visual.js

```diff
@@ -6,9 +6,9 @@ var VisualScheduleCard = {
     should_display_card: function() {
         if (!window.user.student ||
             !window.card_display_dates.is_before_last_day_of_classes) {
```

Comment:
since the schedule includes the final week, should "window.card_display_dates.is_before_last_day_of_classes" be changed?